src/lib/mbtiles.ts

```diff
@@ -46,7 +51,10 @@ export interface Metadata {
    * should be a number, but node-mbtiles implements this as a string, which
    * is the same as TileJSON */
   version?: string
-  json?: string
+  /** Vector layers describe layers of vector tile data */
+  vector_layers?: VectorLayer[]
+  /** Nonstandard tiling scheme added in reference implementation */
+  scheme?: 'xyz' | 'tms'
```

Comment:
see my comment about the scheme field. probably can remove this

Reply:
Dropped in 46053e08923c513c63efc62348aedee84199b039.